cypress/support/api/index.js

```diff
@@ -13,5 +13,9 @@ import './owners';
 import './fees-fines';
 import './organizations';
 import './permissions';
+<<<<<<< HEAD
+import './data-import';
+=======
```

Comment:
Please remove this lines
```suggestion
import './data-import';
```